03_letters/shuutech/main_test.go

```diff
@@ -0,0 +1,53 @@
+package main
+
+import (
+	"fmt"
+	"os"
+	"testing"
+)
+
+func TestLetters(t *testing.T) {
+	var m = letters("aba")
+	var expected int
+	var actual int
+	for key, value := range m {
```

Comment:
due to: https://github.com/anz-bank/go-course/wiki/Lab-2:-Collected-Code-Review-Comments
please `Use table driven tests to avoid repetition (slide sample, Dave Cheney post)`

Reply:
Please also add test cases for an empty string